bbb-exporter/collector.py

```diff
@@ -21,6 +22,19 @@ class BigBlueButtonCollector:
     def __init__(self):
         pass
 
+    def _get_participant_count_by_client(self, meetings):
```

Comment:
This function can be turned static since it is not mutating the internal state of the class. Here is the static method prototype:
```python3
@staticmethod
def _get_participant_count_by_client(meetings):
```

Also this function can be moved bellow `collect()` since its a helper function.